ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+   var body: some View {
+       VStack(spacing: 15) {
+           NavigationStack{
+               VStack(alignment: .leading, spacing: 15){
+                   Text("Task Title")
+                       .multilineTextAlignment(.leading)
+                   TextField("Insert Title", text: $createTaskViewModel.titleInput)
+                       .padding(.leading, 10.0)
+                     .textInputAutocapitalization(.words)
+                     .frame(width: 390.0, height: 40)
+                     .overlay(
+                        RoundedRectangle(cornerRadius: 14)
+                            .stroke(Color.accentColor, lineWidth: 2)
+                     )
+                   Spacer()
```

Comment:
there is no need for this spacer since the gap should be 15, according to Figma. since a spacer is a component of the stack it will have spacing before it and after it, this will result in a visual gap of 45 (spacing + spacer high + spacing). swift ui will automatically take all the available width so just remove it

Reply:
This has been fixed.

Reply:
This has been fixed.